src/cfnlint/data/ExtendedSpecs/all/01_spec_patch.json

```diff
@@ -122,9 +122,18 @@
   },
   {
     "op": "replace",
+    "path": "/ResourceTypes/AWS::IoT::ProvisioningTemplate/Properties/Tags/Type",
+    "value": "List"
+  },
+  {
+    "op": "add",
     "path": "/ResourceTypes/AWS::IoT::ProvisioningTemplate/Properties/Tags/ItemType",
     "value": "Tag"
   },
+  {
+    "op": "remove",
+    "path": "/PropertyTypes/AWS::IoT::ProvisioningTemplate.Tags"
+  },
```

Comment:
just a comment to call this out as manual patching

Reply:
Agreed.  We were already patching it but looks like they made some changes that are still not correct.